permissions/Calendar.permission

```diff
@@ -13,3 +13,7 @@ privileged-data Calendar
 
 mkdir     ${PRIVILEGED}/autofill
 privileged-data autofill
+
+mkdir     ${HOME}/.config/nemo
+whitelist ${HOME}/.config/nemo
```

Comment:
This is suboptimal, as I only want the Calendar app to be able to access the specific file: ${HOME}/.config/nemo/nemo-qml-plugin-calendar.conf

But that directory may or may not include other unrelated config files also...

Any hints?

Reply:
The only thing I can think of would be to move that file to a subdirectory and whitelist that.

Reply:
I think we're going for a /owner/app structure in there too. I already see a com.jolla/calendar directory. Could it be moved to there?

Reply:
Actually, discussed with Pekka that after my PR is merged, we don't actually use the conf file for anything.  Since the calendar widget gets loaded into the homescreen we can rely on it to do the migration, and then we don't need this permission change at all.